harvester/lib/cf_handler.py

```diff
@@ -14,10 +15,10 @@ def setup(self):
         self.client.init_with_user_credentials(self.user, self.password)
         self.task_mgr = TaskManager(self.url, self.client)
 
-    def start_task(self, app_guuid, command, task_id, memory_in_mb=512, disk_in_mb=512):
-        return self.task_mgr.create(
-            app_guuid, command, task_id, memory_in_mb, disk_in_mb
-        )
+    def start_task(self, app_guuid, command, task_id):
+        TASK_MEMORY = os.getenv("HARVEST_RUNNER_TASK_MEM", "4096")
```

Comment:
I can't figure out how to mock these if I put them at the head of the file. I know that gets calculated at import time vs execution time, and [I could use `pytest-env` if I wanted them to be a single, static value](https://github.com/pytest-dev/pytest/discussions/10027), but does anyone on @GSA/data-gov-dev-team know how to monkeypatch them for a single test?

Reply:
does memory and disk space need to derive from the env? could they be default args similar to what you had previously?

Reply:
i'd like them to be configurable, at least while we are trying to nail down what the correct values should be.